<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>421</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="Createtable1">
       <property name="text">
        <string>Создать первую таблицу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Createtable1line">
       <property name="placeholderText">
        <string>Введите количество точек</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>201</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="Createtable2">
       <property name="text">
        <string>Создать вторую таблицу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Createtable2line">
       <property name="placeholderText">
        <string>Введите количество точек</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="Delbutton1">
       <property name="text">
        <string>Удалить строку из первой таблицы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Delline1">
       <property name="placeholderText">
        <string>Введите номер строки для удаления</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QPushButton" name="Delbutton2">
       <property name="text">
        <string>Удалить строку из второй таблицы</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Delline2">
       <property name="placeholderText">
        <string>Введите номер строки для удаления</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Calcbutton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Посчитать результат</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>221</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QTableWidget" name="Table1"/>
     </item>
     <item>
      <widget class="QTableWidget" name="Table2"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QPushButton" name="Adddot1button">
       <property name="text">
        <string>Добавить точку в первую таблицу</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="Dotx1line">
         <property name="placeholderText">
          <string>точка x</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Doty1line">
         <property name="placeholderText">
          <string>точка y</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QPushButton" name="Adddot2button">
       <property name="text">
        <string>Добавить точку во вторую таблицу</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="Dotx2line">
         <property name="placeholderText">
          <string>точка x</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Doty2line">
         <property name="placeholderText">
          <string>точка y</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Detailedbutton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Детальная информация</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>682</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
